src/__tests__/createStore.test.js

```diff
@@ -75,4 +75,55 @@ describe('createStore', () => {
       expect(localStorage.getItem('browserstore_foo')).toBeNull()
     })
   })
+
+  const getMock = jest.fn((error, key) => {
+    if (key !== 'error') throw error
```

Comment:
I don't think this is necessary to conditionally throw. We want to test errors, so we don't have to test for when there's no error (previous test would fail if there is anyway).